tests/test_version.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+from tagbotgitlab.version import get_version_number
+
+
+# Make sure version matches SemVar
+def test_version_string():
+    # Regex found here: https://github.com/k-bx/python-semver/blob/master/semver.py
+    regex = (
+        r"^(?P<major>(?:0|[1-9][0-9]*))\."
+        r"(?P<minor>(?:0|[1-9][0-9]*))\."
+        r"(?P<patch>(?:0|[1-9][0-9]*))"
+        r"(\-(?P<prerelease>(?:0|[1-9A-Za-z-][0-9A-Za-z-]*)"
+        r"(\.(?:0|[1-9A-Za-z-][0-9A-Za-z-]*))*))?"
+        r"(\+(?P<build>[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*))?$"
+    )
```

Comment:
Consider just using `VERBOSE` here. I also pulled the official regex as provided by: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
```python
regex = re.compile(r"""
    ^(?P<major>0|[1-9]\d*)\.
    (?P<minor>0|[1-9]\d*)\.
    (?P<patch>0|[1-9]\d*)
    (?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)
    (?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?
    (?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
""", re.VERBOSE)
```

Reply:
Looks like the same regex to me? Except `\d` is being replaced by `[0-9]`
the VERBOSe is a good idea. I should add that to the cookiecutter